extension/src/popup/views/ManageAssets/index.tsx

```diff
@@ -29,7 +31,13 @@ export const ManageAssets = () => {
     default:
   }
 
-  const { balances } = accountBalances;
```

Comment:
I realized we're loading the source trustlines always even for path payment destination (oops), but fixed now :)